kairon/shared/account/activity_log.py

```
@@ -32,6 +33,29 @@ def add_log(a_type: UserActivityType, account: int = None,  email: Text = None,
+        :param data: dictionary containing data
+        """
+        try:
+            user_activity_log = UserActivityLog(
+                type=a_type,
+                user=email,
+                message=message,
+                data=data
+            )
+            user_activity_log.save()
+        except Exception as e:
+            raise AppException(str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)